packages/core/src/styled/functions/units.ts

```diff
@@ -1,5 +1,10 @@
-const getBaseFs = () =>
-  parseInt(window.getComputedStyle(document.documentElement).fontSize, 10);
+const getBaseFs = () => {
+  if (typeof window !== 'undefined')
```

Comment:
Shouldn't there be an alternative for when there is no window? Or it does not apply?

Reply:
Yeah, I don't know what to do about that when there's no user agent&hellip;

@soslayando, ideas?

Reply:
I have no idea... maybe we have to talk with the people who asked for this feature...

Reply:
Yes @guceda @soslayando ! these conditions are needed to allow SSR to work properly with genesys, as @yairkukielkazunzunegui pointed out.

Otherwise, we will face `runtime` errors while rendering these components on the server-side. We should maintain these conditions to secure third parties can use our components on both client-side apps (SPAs) or server-side apps (SSR).

These changes were introduced by this [MR](https://gitlab.devotools.com/engineering/libraries/coco/-/merge_requests/1242).

Reply:
To clarify, this `if` statement avoid us to execute code that relies on `window` DOM object on the server.

As `node.js` environments don't have a `window` object defined 👀, just DOM clients (browsers) have this.

Reply:
I see. Thank you for weighing in and the explanations.

We may need to review all instances of `window` more thoroughly to make sure something is _returned_ or _done_ server-side when there's not `window` to speak of, then.

I suggest @DevoInc/quvis we file a ticket just for that. In the meantime, we can merge this PR, since it was just to make sure we moved some recent changes correctly from Coco to Genesys UI.

WDYT, @soslayando?